lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -184,7 +185,7 @@ data Depth
 data Role
     = UtxoExternal
     | UtxoInternal
-    | MutableAccount
+    | Stake
```

Comment:
This will cause a breaking change in the API which generically derive path parameters from these constructors. I agree with the change here, but the JSON instance of the API should be modified to also still accept `mutable_account` if provided to not break existing integrations.

Reply:
after some thinking I reverted this change